### DISCUSSION TITLE
Configure Settings for the Browser

### DISCUSSION THREAD

**Discussion by NexGen-3D-Printing** [guysoft/FullPageOS#598] [Q&A]
Hi,

I would like to know how I can change settings for the browser, as nothing seems to work, pressing CTRL+T and changing settings in that Tab will not carry over to the auto loading page.

How can I at least enable dark mode? This information does not help -> https://stackoverflow.com/questions/73606285/how-to-force-css-prefer-color-scheme-to-dark-globally-in-all-chrome-pages I have no idea where to apply the flag as nothing is documented, I have tried and searching through the wiki, issues page and here, so I'm left to ask unfortunately.

**Comment by guysoft** [comment]
Depends on the setting. I think it might carry if you refresh the page with
ctrl+shift+R on the kiosk.

But it might be best to ask the chromium community. Since I dont know all
of chromium's quirks


> .Message ID: ***@***.***>
>

**Reply by NexGen-3D-Printing** [reply]
Thanks for the reply, but I tried that already, didn't work.

Were are all the files located on the system? I assume there is a start script somewhere?

**Reply by guysoft** [reply]
The start script is at:
[/opt/custompios/scripts/start_chromium_browser](https://github.com/guysoft/FullPageOS/blob/devel/src/modules/fullpageos/filesystem/opt/custompios/scripts/start_chromium_browser)
Make sure to run the nightly 0.14.0 which I plan to release soon
